wwwroot/js/site.js

```diff
@@ -45,7 +45,7 @@ function getIPAddress() {
     .then(response => response.json())
     .then(data => {
       let ipString = "Accessing site from: ";
-      if (data["ipAddress"] == "##PROXY_IP##") {
+      if (data["ipAddress"] == "##PROXY_IP##" || data["ipAddress"] == "172.17.0.1") {
```

Comment:
Why are we hardcoding 172.17.0.1?

Reply:
With the move to putting the proxy into the container, it now can come to the webapp as either 10.0.0.5 (usually VM IP) or 172.17.0.1 (always Docker bridge gateway IP).